docs/whitepapers/shortpaper.mdx

```diff
@@ -0,0 +1,641 @@
+
+Nominating a validator also holds a level of risk. If the validator behaves poorly, nominators and the validator will either have a portion of the bond slashed, or a portion of their potential rewards.
+
+If a nominator chooses to unbond their token, there is a cooldown period of 7 days. During this period they will not receive any bonding validator rewards and they will not be able to transfer their token.
+
+#### Validator Estimated ROI
+<table>
+    <tr>
+        <td>Year
+        </td>
+        <td>Revenue (DHX)  \
+            Per Validator \
+            Per Year \
+            (Excluding MSB & MLB)
+        </td>
+        <td>Max. Revenue (DHX) \
+            Per Validator Per Year (Including Max MSB of 60% & Max MLB of 20%
+        </td>
+        <td>Cost (DHX)
+            <p>
+                Per Validator Per Year (Excluding MSB & MLB)
+            </p>
+        </td>
+        <td>ROI
+            <p>
+                (Profit Factor)
+                <p>
+                    Per Validator Per Year (Excluding MSB & MLB)
+                </p>
+            </p>
+        </td>
+        <td>ROI \
+            (Profit Factor) \
+            \
+            Per Validator Per Year
+            <p>
+                (Including MSB & MLB)
+            </p>
+        </td>
+    </tr>
+    <tr>
+        <td>2020 - 2023
+        </td>
+        <td>1752
+        </td>
+        <td>2803.2
+        </td>
+        <td>120
+        </td>
+        <td>14.6
+        </td>
+        <td>23.36
+        </td>
+    </tr>
+    <tr>
+        <td>2024-2027
+        </td>
+        <td>876
+        </td>
+        <td>1401.6
+        </td>
+        <td>120
+        </td>
+        <td>7.3
+        </td>
+        <td>11.68
+        </td>
+    </tr>
+    <tr>
+        <td>2028 - 2031
+        </td>
+        <td>438
+        </td>
+        <td>700.8
+        </td>
+        <td>120
+        </td>
+        <td>3.65
+        </td>
+        <td>5.84
+        </td>
+    </tr>
+</table>
+*Table 1: Validator Estimated Combined ROI*
+
+###Proof of Participation
+#### The 10:1 Rule
+Mining DHX through Proof of Participation requires you to take an active role in DHX governance. To accomplish this, in order to mine DHX, you must first have DHX bonded to a governance council member. This bonded DHX functions as the “limiter” to your mining potential. In order to mine 1 DHX in one day, you need to have both enough mPower and 10 DHX bonded. The bonded DHX must also go through a 7 day cooling-off period. This means that to mine 1 DHX per day for a week, you would need to have 70 DHX bonded.
```

Comment:
nitpick, i noticed we're using “ instead of ", and elsewhere ’ instead of ', but i'm not sure if it matters as it's not code

Reply:
We can leave as is for the moment (time). But yes would be good to clean it up.